src/Repository/CollectorRepository.php

```diff
@@ -63,7 +63,8 @@ private function loadData(): array
         $dataFiles = \glob($this->path . '/yii-debug*.data.json', GLOB_NOSORT);
         $data = [];
         foreach ($dataFiles as $file) {
-            $data = Json::decode(file_get_contents($file));
+            $id = \basename($file, '.data.json');
```

Comment:
Duplicated with loadIndexData. I suggest we leave one method.

```php
private function loadData(string $fileSuffix): array
{
    clearstatcache();
    $dataFiles = \glob($this->path . '/yii-debug*' . $fileSuffix, GLOB_NOSORT);
    $data = [];
    foreach ($dataFiles as $file) {
        $id = \basename($file, $fileSuffix);
        $data[$id] = Json::decode(file_get_contents($file));
    }

    return $data;
}
```

```php
$this->loadData('.index.json');
$this->loadData('.data.json');
```

Reply:
In this case we also need a filename template to be a second parameter for the method `getData()`